code/Video.php

```diff
@@ -44,6 +44,14 @@ class Video extends Page
     private static $default_sort = 'Title ASC';
```

Comment:
If we'll be sorting by the default `SiteTree` sort column, we'll need to remove this static.